agora/main_ui/operations/pushImages.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash
```

Comment:
I think we dont need this - We will have a pipeline that builds and pushes everything to MCR

Reply:
We dont need it in the final version, but its useful for dev. I can just keep it locally in my machine and remove here.